serial/serial_resources/process_entry.py

```diff
@@ -0,0 +1,149 @@
+import serial
+import os
+import mimetypes
```

Comment:
This infers mimetypes from file extensions - left a comment about a package that tries to do the more powerful inference by data.